content/en/docs/tasks/observability/distributed-tracing/zipkin/multicluster_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154,SC2155,SC2034
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# @setup profile=none
+# @multicluster
```

Comment:
I think `multicluster` should be part of the `@setup` config.
```suggestion
# @setup profile=none multicluster
```

Reply:
Would it make sense to setup multiple clusters, all with some profile other than none? If not, maybe we don't need to specify `profile=none` in the setup:
```suggestion
# @setup multicluster
```

Reply:
Agreed. `profile` is per istio installation. With multicluster, we can also have multiple installations. So does profile makes sense? Even if it does, there are many possibilities. To keep things simple, we start with `none`